front/app/containers/EventsShowPage/components/MetadataInformation/Location.tsx

```diff
@@ -56,7 +44,9 @@ const Location = ({ event }: Props) => {
                 fontSize="m"
                 buttonStyle="text"
                 onClick={() => {
-                  setMapModalVisible(true);
+                  window.open(
```

Comment:
Can we do a link here instead of a button? It makes more sense given the functionality.

Reply:
@IvaKop Does the Link component work with external URLs? When I use it, it just appends the Google Maps URL to the platform url (like it's an internal route)? :thinking: 

Code:
```
              <Link
                to={`https://www.google.com/maps/search/?api=1&query=${position.coordinates[1]},${position.coordinates[0]}`}
                target="_blank"
              >
``` 

![image](https://github.com/CitizenLabDotCo/citizenlab/assets/33987955/58f7bebb-7910-48a0-8e69-de6a17e3aa50)

Reply:
You are right. Our Button from the main app, not the component library one, can handle links. Maybe we can use it here?